apps/website/src/pages/institute/pixels.tsx

```diff
@@ -68,12 +68,15 @@ const InstitutePixelsPage: NextPage = () => {
 
   const [fullscreen, setFullscreen] = useState(false);
   const fullscreenRef = useRef<HTMLDivElement>(null);
+  const [scrollProgress, setScrollProgress] = useState(0.5);
+
   const fullscreenToggle = useCallback(() => {
     setFullscreen((v) => {
       if (v) {
         document.body.style.overflow = "";
       } else {
         document.body.style.overflow = "hidden";
+        setScrollProgress(0.5);
```

Comment:
Is this needed, or will the logic below setting the scroll position feed back into this by itself?